.pre-commit-config.yaml

```diff
@@ -55,3 +55,8 @@ repos:
     hooks:
       - id: ruff
         args: [--fix]
+
+  - repo: https://github.com/teemtee/tmt.git
+    rev: 1.23.0
```

Comment:
For 3rd party checks, I'm all for freezing their versions. When it comes to tmt and this kind of dogfooding, is there a risk of using `main`?

Reply:
Would be nice, right? But: 
> [WARNING] The 'rev' field of repo 'https://github.com/teemtee/tmt.git' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.

Reply:
Aaaah... Ok, thanks. Nevermind then.